apps/settings/js/panels/usb_storage/usb_transfer.js

```diff
@@ -71,7 +169,8 @@ define(function(require) {
       } else if (mode === this.MODE_MTP &&
         protocol === this.PROTOCOL_UMS) {
         if (this._partialUmsSupport) {
-          return DialogService.show('partial-ums-warning').then((result) => {
```

Comment:
yeah we can remove return promise here, since its the last item of the caller chain.